assets/app/views/directives/environment.html

```diff
@@ -3,10 +3,11 @@
     <tbody>
       <tr ng-repeat="env in envVars">
         <td>{{env.name}}</td>
-        <td>
+        <!-- Avoid awkward wrapping of long, unbroken values like PATH. -->
+        <td style="word-break: break-all;">
```

Comment:
The mixin wraps at the first `-`. It uses `word-break: break-word;` for Chrome which gives the result as in the first screenshot :(